downloader.py

```diff
@@ -31,13 +31,13 @@ def main(argv):
 	email = ''
 	password = ''
 	directory = 'packtpub_media'
-	formats = 'pdf,mobi,epub,jpg'
+	fileTypes = 'pdf,mobi,epub,jpg,code'
 	includeCode = False
```

Comment:
I didn't add "code" to the default list, as  "--include-code" is also not activated as default.